woocommerce-gateway-ebanx/woocommerce-gateway-ebanx.php

```diff
@@ -121,6 +121,15 @@ public function setup_configs() {
 			$this->is_sandbox_mode = $this->configs->settings['sandbox_mode_enabled'] === 'yes';
 			$this->private_key = $this->is_sandbox_mode ? $this->configs->settings['sandbox_private_key'] : $this->configs->settings['live_private_key'];
 			$this->public_key = $this->is_sandbox_mode ? $this->configs->settings['sandbox_public_key'] : $this->configs->settings['live_public_key'];
+
```

Comment:
The checking is ok, but I think that we can improve this, for example, we have another better places to put the checking, please, take a look on the method `check_environment`, or we can create/rename some methods. In the future we can have more messages like that. What do you think? 😉